source/reference/method/db.cloneCollection.txt

```diff
@@ -26,3 +26,8 @@ Behavior
 --------
 
 .. include:: /includes/extracts/cloneCollection-behavior-method.rst
+
+Example
+-------
```

Comment:
Sorry -- just saw this single sourcing.  We're showing the command example instead of the method example here.  I don't think we can single source this.